src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsTableViewCell.swift

```diff
@@ -280,18 +273,21 @@ class HomeStatisticsTableViewCell: UITableViewCell {
 	private var localStatisticsRegion: LocalStatisticsRegion?
 	private var localStatisticsCache: LocalStatisticsCaching?
 
+	private var wasAlreadyShown = false
+
 	/// Keeping `editingStatistics` locally would reset it on reloading of this cell.
 	/// Terrible design but simpler to handle than states passed through n layers of models, view controllers and views…
-	private static var editingStatistics: Bool = false
+	static var editingStatistics: Bool = false {
+		didSet {
+			()
```

Comment:
Looks strange to me. What does that beautiful masterpiece wants to achieve?

Reply:
It just takes the time get a breath of fresh air and do a whole lot of nothing whenever the editStatistics variable is set 😅

Removed it, just made it easier to observe changes on this variable ^^